src/utils/strhash/city.c

```diff
@@ -36,9 +36,15 @@
 
 #include <stdlib.h>
 #include <stdint.h>
-#include <byteswap.h>
 #include <string.h>
 
+#if defined(__APPLE__)
```

Comment:
Why don't we use __APPLE__ in other places then?

Reply:
This TU has no dependencies on the rest of the system, so we fallback to the "raw" way of detecting OS-dependent things. Otherwise, for everything that depends on `uj_arch`, `UJ_*` constants are used. Personally I find this approach useful because it is hard to remember "proper" patterns of detecting OSes.

Reply:
Ok, agreed